modules/scheduler/pre-existing-gke-cluster/templates/network-object.yaml.tftpl

```diff
@@ -9,3 +9,4 @@ spec:
     kind: GKENetworkParamSet
     name: ${name}
   type: Device
+  provider: GKE
```

Comment:
RDMA on GKE user guide includes this parameter. Trying to understand its significance and if it would cause regression for the multivpc

Reply:
confirmed that this is not required. removing